src/summersplash2022/index.module.css

```diff
@@ -1142,204 +1142,96 @@
   }
 }
 
-@media (min-width: 2000px) {
-  /* 900px*/
-  .scrollContainer,
-  .section1,
-  .section2,
-  .section3,
-  .section5,
-  .whyWorkAtVariant,
-  .section6 {
-    height: auto;
-    scroll-behavior: smooth;
-  }
-
-  .scrollContainer {
-    scroll-behavior: smooth;
-    scroll-snap-type: y mandatory;
-    overflow-y: scroll;
-  }
-
-  /* Headings */
-  .section3Heading,
-  .section5Heading,
-  .section6Heading,
-  .section4Heading,
-  .mostImportantForYou {
-    font-family: 'Recoleta';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 200%;
-    line-height: 39px;
-  }
-
-  /* Section 1 */
-  .imageBlob1 {
-    position: relative;
-    left: 30%;
-    height: auto;
-    bottom: 220px;
-    width: 50%;
+@media (min-width: 2150px) {
+  .heading,
+  .handbookHeaderDeskotop {
+    font-size: 300%;
+    line-height: 200%;
   }
 
-  .newVariantsSVG {
-    width: 70%;
-    height: auto;
+  .handbookHeaderDeskotop {
+    padding: 0px;
   }
 
-  .søkHer,
-  .scroll {
-    font-family: 'Graphik Web';
-    font-style: normal;
-    font-weight: 500;
+  p,
+  .section2Paragraph,
+  .whyWorkAtVariant p,
```

Comment:
En annen ting jeg ville prøvd å hatt i bakhodet til senere er at variant-prosjekter generelt prøver å bruke [BEM-modellen](http://getbem.com/naming/) til navngivning av html-klasser. Tror dette er ganske vanlig generelt i IT-verdnen, så uansett nyttig å lære seg. Jeg er fortsatt der at jeg ikke er helt vant til å bruke det, men prøver å være oppmerksom på det hvertfall.

Ville definitivt ikke gått tilbake og endret i dette prosjektet tho, både fordi det er temporary og fordi det er mye jobb for ganske liten gevinst når det ikke skal jobbes noe særlig på videre.